### ISSUE TITLE
Extension not working

### ISSUE THREAD

**Issue by kanlanc** [vladmrnv/ExpressSnippet#9]
I was a very extensive user of this extension but I don't know maybe because of the update to VS code or something this extension completely stopped showing any snippet. I uninstalled and installed it multiple times but it did not solve the problem. Please look into this

**Comment by vladmrnv** [comment]
Oh wow you're right, I'll try to push an update tonight and see what the issue is. Thanks for informing me.

**Comment by Bengals10** [comment]
UP

Still not working!

**Comment by dizthewize** [comment]
Agreed it stopped working for me as well and this is a very useful extension
